include/misc/print_table.hpp

```diff
@@ -0,0 +1,381 @@
+// Header file from: https://github.com/friedmud/variadic_table/blob/master/include/VariadicTable.h
```

Comment:
ok the license are compatible, but please add the project header above and then we could ack the source of the code

Reply:
Resolved.